Makefile

```diff
@@ -27,8 +27,12 @@ all: bootstrap
 config:
 ifeq (riscv,$(ARCH))
 	@echo $(RISCV_EXEC) > $(OUT)/target
+	@echo \#define TARGET_RISCV 1 > config.h
+	@ln -s riscv-codegen.c $(SRCDIR)/codegen.c
```

Comment:
Should be `$(SRCDIR)/riscv-codegen.c`.

Reply:
`$(SRCDIR)/riscv-codegen.c` will fail to include `codegen.c`, so I change to `riscv-codegen.c` and it works good.

Reply:
You should figure out why it failed.

Reply:
How about this:
```shell
ln -s $(PWD)/$(SRCDIR)/arm-codegen.c $(SRCDIR)/codegen.c
```
It also works good.